src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml

```diff
@@ -69,6 +70,7 @@
                     </div>
                 </div>
             }
+            <a asp-controller="InviteUser" asp-action="Invite" class="button">Invite new user</a>
```

Comment:
This will show the button correctly when the css upgrade is merged:
```suggestion
            <a asp-controller="InviteUser" asp-action="Invite" class="button govuk-button">Invite new user</a>
```